docs/src/app/components/DatePicker/README.md

```diff
@@ -0,0 +1,3 @@
+## Date Picker
+
```

Comment:
Can you add the description to https://www.google.com/design/spec/components/pickers.html#pickers-date-pickers?